beakerx-base/LICENSE

```diff
@@ -0,0 +1,202 @@
+Apache License
```

Comment:
we don't need any more copies of this.  the one at the top of the repo is enough.